rclcpp/test/rclcpp/test_parameter.cpp

```diff
@@ -32,7 +32,7 @@ class TestParameter : public ::testing::Test
   }
 };
 
-TEST(TestParameter, not_set_variant) {
+TEST_F(TestParameter, not_set_variant) {
```

Comment:
the TEST -> TEST_F changes are a bit unrelated, and it would be great to do them in a separate PR.
I don't mind accepting them here though